docs/sources/experiments/cohort-labels.md

```diff
@@ -0,0 +1,493 @@
+# Cohort and Label Deep Dive
+
+This document is intended at providing a deep dive into the concepts of cohorts and labels as they apply to Triage. For context, reading the [Triage section of the Dirty Duck tutorial](https://dssg.github.io/dirtyduck/#sec-4-2) may be helpful before reading this document.
+
+## Temporal Validation Refresher
+
+Triage uses temporal validation to select models because the real-world problems that Triage is built for tend to have a strong temporal correlation.  Picking a date range to train on and a date range afterwards to test on ensures that we don't leak data from the future into our models that wouldn't be available in a real-world deployment scenario. Because of this, we often talk in Triage about the *as-of-date*: all models trained by Triage are associated with an *as-of-date*, which means that all the data that goes into the model **is only included if it was known about before that date**. For more on temporal validation, see the [relevant section in Dirty Duck](https://dssg.github.io/dirtyduck/#sec-4-2-2-1).
+
+## What are Cohorts and Labels in Triage?
+
+This document assumes that the reader is familiar with the concept of a machine learning target variable and will focus on explaining what is unique to Triage.
+
+**A cohort is the population used used for modeling on a given as-of-date**. This is expressed as a list of *entities*. An entity is simply the object of prediction, such as a facility to inspect or a patient coming in for a visit. Early warning systems tend to include their entire population (or at least a large subset of it) in the cohort at any given date, while appointment-based problems may only include in a date's cohort the people who are scheduled for an appointment on that date.
```

Comment:
list -> set

Reply:
"Early warning systems tend to include their entire population (or at least a large subset of it)" -> "EWS tend to include their entire population"

Reply:
I was trying to make the DDJ-type early warning cohorts fit here. The cohorts are only sometimes the entire population but can sometimes be like very large subsets like 'people who have ever been arrested'. I wanted a sentence that draws a distinction between visit-level problems that have a small per-date cohort vs everything else

Reply:
Might be illustrative to give a quick example of each case. I think we have a good handle on the difference between those two types of problems, but it will probably be less familiar to external readers.